contracts/polygon/connectors/aave/v3-import/main.sol

```diff
@@ -105,6 +187,21 @@ contract AaveV3ImportResolver is AaveHelpers {
 	{
 		(_eventName, _eventParam) = _importAave(userAccount, inputData);
 	}
+
+	/**
+	 * @dev Import aave V3 position (with collateral).
+	 * @notice Import EOA's aave V3 position to DSA's aave v3 position
+	 * @param userAccount The address of the EOA from which aave position will be imported
+	 * @param inputData The struct containing all the neccessary input data
+	 * @param enableCollateral The boolean array to enable selected collaterals in the imported position
+	 */
+	function importAaveWithCollateral(address userAccount, ImportInputData memory inputData, bool[] memory enableCollateral)
+		external
+		payable
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		(_eventName, _eventParam) = _importAaveWithCollateral(userAccount, inputData, enableCollateral);
+	}
 }
 
 contract ConnectV2AaveV3ImportPolygon is AaveV3ImportResolver {
```

Comment:
Update version on name to `Aave-v3-import-v1.1`

Reply:
done